### DISCUSSION TITLE
Game And Feature Examples For Inspiration

### DISCUSSION THREAD

**Discussion by WithinAmnesia** [SlayHorizon/godot-tiny-mmo#17] [Show and tell]
Here is a list of games and features that can be excellent inspiration:

Open Source Inspirations:
- Flare (Flare Engine and Flare Game)
- TetraForce (Godot 3)
- Solaris (Book of Mudora)
- Voxelgame (Blocky Game. 3D Godot 4.XX multiplayer voxel game demo)
- Creative Commons 0 licensed OpenGameArt Assets (CC0)
- Creative Commons 0 licensed Itch.io Assets (CC0)
- Wyvern Systems (Godot Wyvernshield and Wyvernbox)
- JDungeon (With Class Support)
- The Soulhunter (Godot 3)
- Explosive Salad (3D Godot 4.XX multiplayer fps game demo) https://github.com/TriggerCoder/Explosive-Salad

Non-Open Source Inspirations:
- World of Warcraft Vanilla / Classic (Turtle WoW, Vanilla Kronos WoW, Vanilla+ and WotLK private servers)
- Baldur's Gate (1, 2 and 3)
- Diablo (1 and 2)
- The Legend of Zelda (Main series releases)
- Mount and Blade (1 and 2)
- Minecraft (Java)
- Dungeons and Dragons (1st, 2nd, 3rd and 5th Editions)
- Lord of the Rings (Source-based Tolkien's Works)
- Rainbow Fairy Books (Lang's Rainbow Fairytale Books)
- Game of Thrones (A Song of Ice and Fire)

**Comment by WithinAmnesia** [comment]
This is cool but its not very open source, it's almost free but its not for game making tools but its cool to see: https://deepdivegamestudio.itch.io/animalassetpack

Maybe if we use 2D sprites and the flip back and forth (as they currently do) the animation resources can literally be 1 frame of art for simple things that can move up and down on a placeholder animation template. This would enable tonnes of art and make game making the most simple possible.

I'll try to find even more CC0 / MIT free art / animation resources too.

**Comment by SlayHorizon** [comment]
I don't think most of the resources here are compatible with the current project.
For the arts, as you said, they are not allowed in such a project, but anyway, I don't think we need anything else for what we want to achieve currently, unless maybe some custom cursors like these ones [https://kenney.nl/assets/cursor-pack](https://kenney.nl/assets/cursor-pack).

**Reply by WithinAmnesia** [reply]
I was thinking about the art format that is currently used in the demo and how its super handy to make working characters out of a lot less art and its a really good base to get most people going faster. It can be expanded / made more advanced later too. I like how simple and effective the current art sprite format is as where I realized it can use 4/5 to 9/10+ or the 2D character art. As in this current art format can achieve big production value character controls / gameplay with small budget for art and animations.

The simpler art animations but still deep character range of motion potential move set with the current character animations in the demo help free people from needing tons of animations. Which is great for most people can make cool games without needing tons of animations resources / creativity. So most everyone can focus on building their game(s) with low input and get high output which is great for starting as independent game developers.

I can help with icons and I'll look for cursors and just make a list or a nice to have art in X or Y wanted formats (like we could use cursors or icons or GUI etc.) I'll look for some in the big Creative Commons 0 / public domain asset archive I have been compiling for the creative community. Its really good you got the art from the Dungeon Crawler to be used here too to help the community with MIT and open source. MIT is super good like C.C.0 for helping people start making cool games. I agree with you on that those assets I liked above are not compatible with the project too. I'll try to find C.C.0 / MIT / useful art that is compatible and see what people think too / get feedback / maybe open a discussion / show and tell / Q & A on if we should use some more C.C.0 / MIT art. Also I'll help source art and audio and any C.C.0 / MIT assets what we could use right away to help in the immediate situation. All feedback, ideas and questions are welcome.

**Comment by WithinAmnesia** [comment]
https://github.com/SlayHorizon/godot-tiny-mmo-demo/discussions/27#discussioncomment-10791945
<details>
<summary>List of cool things, ideas, inspirations and sharing community world building section</summary>
"In ways I sort of miss that warmth of being around the awesome passionate fire of creation within the creative community. Yet perhaps in time things can organically grow back with a demo or two or few or more. Perhaps I should make a demo / world building section and pour my heart and soul into that and give some room to breathe, share and perhaps find more like-minded creative people along the way. The community loves building a world and letting the games and universes sort of tell themselves and create new adventures."

From: https://github.com/SlayHorizon/godot-tiny-mmo-demo/discussions/27#discussioncomment-10791945

</details>
This is cool:

World of Warcraft on the NES?! (2D MMORPG Visualization)

[https://www.youtube.com/watch?v=64oDMp0FGxA](https://www.youtube.com/watch?v=64oDMp0FGxA)
